addons/mail/models/mail_template.py

```diff
@@ -30,7 +30,9 @@ def default_get(self, fields):
     model_id = fields.Many2one('ir.model', 'Applies to', help="The type of document this template can be used with")
     model = fields.Char('Related Document Model', related='model_id.model', index=True, store=True, readonly=True)
     subject = fields.Char('Subject', translate=True, help="Subject (placeholders may be used here)")
-    email_from = fields.Char('From',
+    author_id = fields.Char('From (User)', placeholder="Author user's id, override the author",
```

Comment:
Why is author a Char field, and not a Many2one pointing to a res.users ?

Reply:
Well it's purpose is notably to hold jinja code ... like object.user_id.partner_id.id to simulate that the author of a message is the responsible of a document. That's why it is a char field, like partner_to for example. What has to be taken into account is that it may hold incoherent values, casting into an int should be done with care.

Maybe author_id is not a good name note, simply author maybe ? To avoid telling it should be a m2o

What do you think @jeh-odoo ?

Reply:
@tde-banana-odoo Yes, I'm ok with renaming it author. But the message of @tivisse gave me an idea to get rid of the increase of the number of queries.